monitoring/uss_qualifier/scd/executor/test_steps/inject_flight.py

```diff
@@ -18,6 +19,15 @@
 def execute(runner, step: TestStep, step_ref: TestStepReference, target: TestTarget) -> None:
     print("[SCD]     Step: Inject flight {} to {}".format(step.inject_flight.name, target.name))
     try:
+        reference_time = datetime.utcnow() + timedelta(seconds=pytimeparse.parse(step.inject_flight.planning_time))
+        step.inject_flight.reference_time = reference_time.isoformat() + "Z"
+        for volume in step.inject_flight.test_injection.operational_intent.volumes:            
+            time_start = reference_time + timedelta(minutes=3)
```

Comment:
I would not expect any magic numbers (3 minutes and 5 minutes in this case) in the adjustment algorithm